app/uk.gov.hmrc.helptosaveproxy/config/MicroserviceWiring.scala

```diff
@@ -80,21 +65,20 @@ class WSHttpExtension extends WSHttp with HttpAuditing with ServicesConfig {
 
 }
 
-class WSHttpProxy @Inject() (config: Configuration, proxyConfigurationPath: String)
+@Singleton
+class WSHttpProxy @Inject() (override val auditConnector: AuditConnector, config: Configuration)
   extends HttpPost with WSPost
   with HttpPut with WSPut
   with HttpGet with WSGet
   with WSProxy
-  with RunMode
   with HttpAuditing
   with HttpVerbs {
 
   val httpReads: HttpReads[HttpResponse] = new RawHttpReads
 
   override lazy val appName: String = config.underlying.getString("appName")
-  override lazy val wsProxyServer: Option[WSProxyServer] = WSProxyConfiguration(proxyConfigurationPath)
+  override lazy val wsProxyServer: Option[WSProxyServer] = WSProxyConfiguration("microservice.services.dwp.proxy")
```

Comment:
this will always pick up dwp's proxy configuration - the `proxyConfigurationPath` was there before so that we can have separate configuration for NS&I and DWP